Sources/Quick/World.swift

```diff
@@ -44,13 +44,7 @@ final internal class World: NSObject {
         within this test suite. This is only true within the context of Quick
         functional tests.
     */
-#if _runtime(_ObjC)
-    // Convention of generating Objective-C selector has been changed on Swift 3
-    @objc(isRunningAdditionalSuites)
     internal var isRunningAdditionalSuites = false
-#else
-    internal var isRunningAdditionalSuites = false
-#endif
```

Comment:
Sorry to be pedantic, but I'd prefer if this were in a separate pull request, if for no other reason than we can use the commit message to document why this is no longer needed.